### PR TITLE
LPS-129812 Fix onDoubleClick event

### PR DESCRIPTION
**Pull request by alinedoleron**
Related ticket: https://issues.liferay.com/browse/LPS-129812

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/plugins/field-sidebar/components/FormsFieldSidebar.es.js [hunk 1]
```diff
@@ -112,22 +112,25 @@ export const FormsFieldSidebar = ({title}) => {
...
+							}),
+							editable: true,
+						},
+						indexes: {
+							columnIndex: 0,
+							pageIndex: activePage,
+							rowIndex: pages[activePage].rows.length,
+						},
 					},
+					type: CORE_EVENT_TYPES.FIELD.ADD,
```
Comment (diegonvs) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/plugins/field-sidebar/components/FormsFieldSidebar.es.js:133]: ✅

**Review by diegonvs — APPROVED** [review]
Looks palmas(good) to me!

**Comment by alinedoleron** [comment]
ci:forward

**Pull request closed without merging**